core/src/main/resources/application.properties

```diff
@@ -80,7 +80,14 @@ klaw.enable.sso=false
 # Uncomment the below to establish connectivity between core and cluster apis.
 # Provide a base 64 encoded string. The same secret should be configured in Klaw Cluster Api.
 klaw.clusterapi.access.base64.secret=
-
+# This allows App2App comunication in a HA setup with multiple Klaw-core instances
+# Provide a base 64 encoded string. The same secret should be used in all Instances.
+# Update the below to your own unique secret!!!!!
+klaw.core.app2app.base64.secret=dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==
+klaw.core.app2app.username=KlawApp2App
+#
+# Enable High Availability for multi-instance Klaw-core deployment.
+klaw.core.ha.enable=true
```

Comment:
do we want to enable this by default ?

Reply:
No we don't!! I remember why i did this now thats what enables the openapi.yaml to generate the code for the UI, I'll fix that though great catch!